### DISCUSSION TITLE
UP to 2.4.100,so-telegraf doesn't work at all

### DISCUSSION THREAD

**Discussion by tonyzuqhd** [Security-Onion-Solutions/securityonion#13641] [Unsupported Versions]
Yesterday,I soup my 2.4.90 to 2.40.100,after that,web manager in grid section all the node red alart.when I check the manager-node,found that telegraf doesn't work.I switch to console,so-telegraf-restart,and,get some error report list down.
The list was so long,I just post the error section.
==============================================================================
----------
          ID: so-telegraf
    Function: docker_container.running
      Result: False
     Comment: Failed to pull somanager:5000/security-onion-solutions/so-telegraf:2.4.100: Error 404: manifest for somanager:5000/security-onion-solutions/so-telegraf:2.4.100 not found: manifest unknown: manifest unknown
     Started: 02:24:05.903616
    Duration: 234.488 ms
     Changes:   
----------
          ID: delete_so-telegraf_so-status.disabled
    Function: file.uncomment
        Name: /opt/so/conf/so-status/so-status.conf
      Result: True
     Comment: Pattern already uncommented
     Started: 02:24:06.138969
    Duration: 6.758 ms
     Changes:   

Summary for local
-------------
Succeeded: 93 (changed=1)
Failed:     1
-------------

Is anyone could teach me how to solve this situation?Thanks

**Comment by TOoSmOotH** [comment]
Sounds like it never made it to your local repo. Run so-docker-refresh on your manager.

**Reply by tonyzuqhd** [reply]
Thanks,Checked all the things,and I  reinstall manager node.And,Change the administrator password,The rest parts as idh,searchnode,forwardnode,didn't reinstall.So,new question come,How to Change the rest node's configed password of the manager.

**Comment by tonyzuqhd** [comment]
I tried to change my new manager node linux OS admin's password to fit the old manager's password.It seam to be unuseful .The old idh node could not connect to the new manager node successful.In the firewall has add the address of the old idh,the grid member has add the idh .BUT,the new manager 'grid has no any older friend at all.So,I guest,there must be some security issues in new manager,to prevent the old idh connect to new manager.Maybe ,this is a Security mechanism.

**Reply by dougburks** [reply]
Grid nodes do not use the admin password to connect to the manager. If you reinstall the manager node, then you will have to rejoin each and every node individually. If you have further questions or problems, then please start a new discussion with appropriate title since we're no longer discussing telegraf.
